environments/icds-cas/inventory/citusdb.csv

```diff
@@ -1,62 +1,68 @@
-citusworker18standby,100.71.184.241,citusdb,citusdb_worker,pg_standby,MUMGCCWCDPRDCWV18,,citusworker18,,standby0
-citusworker19standby,100.71.184.242,citusdb,citusdb_worker,pg_standby,MUMGCCWCDPRDCWV19,,citusworker19,,standby0
-citusworker20standby,100.71.184.243,citusdb,citusdb_worker,pg_standby,MUMGCCWCDPRDCWV20,,citusworker20,,standby0
-citusworker21standby,100.71.184.244,citusdb,citusdb_worker,pg_standby,MUMGCCWCDPRDCWV21,,citusworker21,,standby0
-citusworker22standby,100.71.184.245,citusdb,citusdb_worker,pg_standby,MUMGCCWCDPRDCWV22,,citusworker22,,standby0
-citusworker23standby,100.71.184.246,citusdb,citusdb_worker,pg_standby,MUMGCCWCDPRDCWV23,,citusworker23,,standby0
-,,,,,,,,,
-group,var,val,type,,,,,,
-citusdb,ansible_python_interpreter,/usr/bin/python3,S,,,,,,
-citusdb,pgbouncer_max_connections,3500,I,,,,,,
-citusdb,postgresql_archive_command,/bin/true,S,,,,,,
-citusdb_master,pgbouncer_default_pool,790,I,,,,,,
-citusdb_master,postgresql_checkpoint_completion_target,0.9,F,,,,,,
-citusdb_master,postgresql_default_statistics_target,500,I,,,,,,
-citusdb_master,postgresql_effective_cache_size,80GB,S,,,,,,
-citusdb_master,postgresql_effective_io_concurrency,200,I,,,,,,
-citusdb_master,postgresql_maintenance_work_mem,2GB,S,,,,,,
-citusdb_master,postgresql_max_connections,2000,I,,,,,,
-citusdb_master,postgresql_max_parallel_workers_per_gather,4,I,,,,,,
-citusdb_master,postgresql_max_wal_size,8GB,S,,,,,,
-citusdb_master,postgresql_max_worker_processes,8,I,,,,,,
-citusdb_master,postgresql_min_wal_size,4GB,S,,,,,,
-citusdb_master,postgresql_random_page_cost,1.1,F,,,,,,
-citusdb_master,postgresql_shared_buffers,40GB,S,,,,,,
-citusdb_master,postgresql_wal_buffers,16MB,S,,,,,,
-citusdb_master,postgresql_work_mem,1872kB,S,,,,,,
-citusdb_worker,pgbouncer_default_pool,1500,I,,,,,,
-citusdb_worker,postgresql_checkpoint_completion_target,0.9,F,,,,,,
-citusdb_worker,postgresql_default_statistics_target,500,I,,,,,,
-citusdb_worker,postgresql_effective_cache_size,60GB,S,,,,,,
-citusdb_worker,postgresql_effective_io_concurrency,200,I,,,,,,
-citusdb_worker,postgresql_maintenance_work_mem,2GB,S,,,,,,
-citusdb_worker,postgresql_max_connections,2000,I,,,,,,
-citusdb_worker,postgresql_max_parallel_workers_per_gather,16,I,,,,,,
-citusdb_worker,postgresql_max_wal_size,8GB,S,,,,,,
-citusdb_worker,postgresql_max_worker_processes,32,I,,,,,,
-citusdb_worker,postgresql_min_wal_size,4GB,S,,,,,,
-citusdb_worker,postgresql_random_page_cost,1.1,F,,,,,,
-citusdb_worker,postgresql_shared_buffers,20GB,S,,,,,,
-citusdb_worker,postgresql_wal_buffers,16MB,S,,,,,,
-citusdb_worker,postgresql_work_mem,1872kB,S,,,,,,
+hostname,host_address,group 1,group 2,group 3,group 4,var: hostname,H.var: hot_standby_server,H.var: hot_standby_master,L.var: postgresql_replication_slots,var: replication_slot
+citusmaster1,100.71.184.221,citusdb_pg_master,,citusdb_master,,MUMGCCWCDPRDCCV00,citusmaster1standby,,"[""standby0""]",
```

Comment:
Here i have two separate group `citusdb_pg_master` and `citusdb_pg_standby` in place of ` citusdb`